apollo-gradle-plugin-incubating/src/main/kotlin/com/apollographql/apollo/gradle/AndroidTaskConfigurator.kt

```diff
@@ -49,12 +49,11 @@ object AndroidTaskConfigurator {
     )
 
     registerVariantTask(project, apolloVariant) { serviceVariantTask ->
+      variant.registerJavaGeneratingTask(serviceVariantTask.get(), serviceVariantTask.get().outputDir)
       if (apolloExtension.generateKotlinModels) {
+        // For java, this is done by registerJavaGeneratingTask
```

Comment:
Does this mean 'java as opposed to android'?

Reply:
For android "java models". This file is android-only.